api/v1alpha1/sandbox_types.go

```diff
@@ -66,6 +66,9 @@ type SandboxSpec struct {
 	// PodTemplate describes the pod spec that will be used to create an agent sandbox.
 	// +kubebuilder:validation:Required
 	PodTemplate PodTemplate `json:"podTemplate" protobuf:"bytes,3,opt,name=podTemplate"`
+
+	// ShutdownAt - Absolute time (RFC 3339) when the sandbox is deleted.
+	ShutdownAt string `json:"shutdownAt,omitempty"`
```

Comment:
is this immutable ? if not, it would be treated as also, extending the ttl duration right ?

Reply:
There's no immutability enforcement in this PR.